app/instance/.env.example

```
@@ -53,3 +53,18 @@ HOST=0.0.0.0
+MAIL_PASSWORD=<YOUR_PASSWORD_OR_APP_PASSWORD>
+MAIL_FROM=sender@example.com
+MAIL_FROM_NAME="LLM Analyzer"
+MAIL_PORT=587
+MAIL_SERVER=smtp.example.com
+MAIL_STARTTLS=true
+MAIL_SSL_TLS=false
+MAIL_USE_CREDENTIALS=true
+MAIL_VALIDATE_CERTS=true
+MAIL_SUPPRESS_SEND=true
+MAIL_DEBUG=0
+MAIL_AUDIT_RECIPIENTS=[]
```

dotenv-linter (3.3.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)